DES-58 Ordered List (#430)
Amsterdam/design-system: packages/css/src/ordered-list/README.md

# Ordered List

> ## Introductie

Reviewer: This differs from the [component docs guidelines](https://github.com/Amsterdam/design-system/blob/main/documentation/component-docs-guidelines.md), the docs shouldn't have an 'Introductie' heading and 'Specificaties' should be 'Richtlijnen' I think

> Een lijst is een verticale groep gerelateerde inhoud.

Reviewer: This is a little generic, maybe specify that it's hierarchical?

Author: Formatted it more like an introduction paragraph like https://www.amsterdam.nl/digitalservices/design-system/atomen-moleculen/list/

## Specificaties

> Deze lijst kan 2 niveaus hebben. Het eerste niveau is genummerd. Het tweede niveau is in onderkast alfabetische volgorde. Tekst in de lijstitems springt een vaste afstand in.

Reviewer: ```suggestion
Deze lijst kan 2 niveaus hebben. Het eerste niveau is genummerd. Het tweede niveau gebruikt letters in alfabetische volgorde. Tekst in de lijstitems springt een vaste afstand in.
```

## Terminologie

- **ordered list**: uitgeschreven versie van de HTML afkorting `<ol>`

> - **list item**: uitgeschreven van van [HTML `<li>`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element). Ook [in ARIA is het `role="listitem"`](https://www.w3.org/TR/wai-aria-1.2/#listitem)

Reviewer: ```suggestion
- **list item**: uitgeschreven versie van [HTML `<li>`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element). Ook [in ARIA is het `role="listitem"`](https://www.w3.org/TR/wai-aria-1.2/#listitem)
```

Author: Aangepast

# Ordered List

> Een lijst is een verticale groep gerelateerde inhoud. Deze lijst kan 2 niveaus hebben. Het eerste niveau is genummerd. Het tweede niveau gebruikt letters in alfabetische volgorde. Tekst in de lijstitems springt een vaste afstand in.

Reviewer: ```suggestion
Een geordende lijst is een verticale groep gerelateerde inhoud met een volgorde. Deze lijst kan 2 niveaus hebben. Het eerste niveau is genummerd. Het tweede niveau gebruikt letters in alfabetische volgorde. Tekst in de lijstitems springt een vaste afstand in.
```
